stdlib/experimental/durations_test.flux

```diff
@@ -5,6 +5,7 @@ import "csv"
 import "testing"
```

Comment:
As durations have been moved to date from experimental, this test can be removed from experimental.

Reply:
Removed.